build.sbt

```diff
@@ -47,123 +46,109 @@ lazy val commonSettings = Seq(
   pomExtra := pomInfo,
   resolvers += Opts.resolver.mavenLocalFile,
   autoAPIMappings := true,
-  libraryDependencies ++= excludeServlet(Seq(
-    "org.scalaz" %% "scalaz-core" % "7.1.3",
-    "org.mortbay.jetty" % "servlet-api" % "3.0.20100224" % "provided",
-    "org.apache.hadoop" % "hadoop-common" % hadoopVer % "provided",
-    "org.apache.hadoop" % "hadoop-mapreduce-client-core" % hadoopVer % "provided",
-    "org.apache.parquet" % "parquet-column" % parquetVer,
-    "org.apache.parquet" % "parquet-hadoop" % parquetVer
-  )),
   libraryDependencies ++= Seq(
-    "io.circe" %% "circe-core",
-    "io.circe" %% "circe-generic",
-    "io.circe" %% "circe-parser"
-  ).map(_ % circeVersion),
-  dependencyOverrides += "org.slf4j" % "slf4j-log4j12" % "1.7.12",
-  dependencyOverrides += "org.slf4j" % "slf4j-api" % "1.7.12",
+    "org.apache.hadoop" % "hadoop-common" % hadoopVer % "provided,test",
```

Comment:
There are very few _actual_ common dependencies - I have figured out what these actually are and made them the only ones that are common now.

Note that any `provided` dependencies from here onwards are also added to the `test` scope so that when you add a `test->test` dependency on another module, you get it as a dependency.